okhttp-logging-interceptor/README.md

```diff
@@ -0,0 +1,44 @@
+Logging Interceptor
+===================
+
+An [OkHttp interceptor][1] which logs HTTP request and response data.
```

Comment:
Added:

**Warning**: The logs generated by this interceptor when using the `HEADERS` or `BODY` levels has
the potential to leak sensitive information such as "Authorization" or "Cookie" headers and the
contents of request and response bodies. This data should only be logged in a controlled way or in
a non-production environment.